apps/cowswap-frontend/src/modules/ordersTable/pure/OrdersTableContainer/index.tsx

```diff
@@ -12,15 +12,19 @@ import styled from 'styled-components/macro'
 
 import { Web3Status } from 'modules/wallet/containers/Web3Status'
 
+import { UI } from 'common/constants/theme'
 import { CowSwapSafeAppLink } from 'common/pure/CowSwapSafeAppLink'
 
 import { OrdersTable, OrdersTableProps } from './OrdersTable'
 import { OrdersTabs, OrdersTabsProps } from './OrdersTabs'
 
-const OrdersBox = styled.div`
-  background: transparent;
-  border-radius: 16px;
-  padding: 0;
+const OrdersBox = styled.div<{ isWidgetMode: boolean }>`
```

Comment:
We already have `theme.isInjectedWidgetMode`, so we don't need to put a custom prop here

Reply:
Aaah right of course, I forgot about it. I will have to change it in more places I think...